sbin/system-setup.py

```diff
@@ -55,6 +58,7 @@ def common_last(self):
 
 parser = argparse.ArgumentParser(description='Set up system for build.')
 parser.add_argument('-n', '--nop', action="store_true", help='no operation')
+parser.add_argument('--coverage', action="store_true", help='setup for coverage analisys (older CLang version)')
```

Comment:
```suggestion
parser.add_argument('--coverage', action="store_true", help='setup for coverage analysis (older CLang version)')
```

Reply:
I should have removed the option altogether as we no longer check for it.